portlets/pom.xml

```diff
@@ -19,59 +19,172 @@
+            <groupId>org.juzu</groupId>
+            <artifactId>juzu-plugins-jackson</artifactId>
+            <version>${juzu.version}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.exoplatform.core</groupId>
+            <artifactId>exo.core.component.organization.api</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <!-- Commons project-->
+        <dependency>
+            <groupId>org.exoplatform.commons</groupId>
+            <artifactId>commons-juzu</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <!-- Gatein deps-->
+        <dependency>
+            <groupId>org.exoplatform.gatein.portal</groupId>
+            <artifactId>exo.portal.component.web.controller</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.exoplatform.gatein.portal</groupId>
+            <artifactId>exo.portal.webui.portal</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <!-- Categories addon dep -->
+        <dependency>
+            <groupId>org.exoplatform.addons.gamification</groupId>
+            <artifactId>gamification-services</artifactId>
+            <scope>provided</scope>
+        </dependency>
+    </dependencies>
+    <build>
+        <finalName>gamification-portlets</finalName>
+        <plugins>
+            <plugin>
+                <groupId>com.github.eirslett</groupId>
+                <artifactId>frontend-maven-plugin</artifactId>
+                <version>1.5</version>
+                <configuration>
+                    <workingDirectory>src/main/frontend</workingDirectory>
+                </configuration>
+                <executions>
+                    <!-- Install our node and npm version to run npm/node scripts-->
+                    <execution>
+                        <id>install node and npm</id>
+                        <goals>
+                            <goal>install-node-and-npm</goal>
+                        </goals>
+                        <configuration>
+                            <nodeVersion>v9.4.0</nodeVersion>
+                            <npmVersion>5.6.0</npmVersion>
+                            <nodeDownloadRoot>https://nodejs.org/dist/</nodeDownloadRoot>
+                            <npmDownloadRoot>http://registry.npmjs.org/npm/-/</npmDownloadRoot>
+                        </configuration>
+                    </execution>
+                    <!-- Set NPM Registry -->
+                    <execution>
+                        <id>npm set registry</id>
+                        <goals>
+                            <goal>npm</goal>
+                        </goals>
+                        <configuration>
+                            <arguments>config set registry https://registry.npmjs.org</arguments>
+                        </configuration>
+                    </execution>
```

Comment:
Is this command needed ?

Reply:
Not mandatory conf, but I would like to keep it at this stage